client/core/bookie.go

```diff
@@ -883,32 +883,6 @@ func (dc *dexConnection) refreshServerConfig() (*msgjson.ConfigResult, error) {
 		}
 	}
 
-	// Patch ConfigResponse.RegFees if no entry for DCR is there, meaning it is
-	// likely an older server using cfg.Fee and cfg.RegFeeConfirms.
-	if dcrAsset := cfg.RegFees["dcr"]; dcrAsset == nil {
-		dc.log.Warnf("Legacy server %v does not provide a regFees map.", dc.acct.host)
-		if cfg.RegFees == nil {
-			cfg.RegFees = make(map[string]*msgjson.FeeAsset)
-		}
-		if cfg.Fee > 0 {
```

Comment:
Release clients will incorrectly refer to this as a "Legacy server" if the server lacks a DCR entry in the map, but the behavior is otherwise fine (block registering with DCR).